defs/scenarios/storage/ceph/ceph-osd/PG_overdose.yaml

```diff
@@ -0,0 +1,25 @@
+checks:
+  health_warning:
+    property:
+      path: hotsos.core.plugins.storage.ceph.CephCluster.health_status
+      ops: [[ne, HEALTH_OK]]
+  # this one will run on OSD sos
+  ceph_osd_withhold_creation:
+    input:
```

Comment:
i think this should be:

    path: [var/log/ceph/ceph-osd*.log, var/log/ceph/ceph.log]

Reply:
done